### DISCUSSION TITLE
寻找社区微信群/qq群/钉钉群

### DISCUSSION THREAD

**Discussion by zhilaohu32** [apache/doris#16887] [A - General / Q&A]
不知道社区有没有相关交流群，希望能够加入！

**Comment by yagagagaga** [comment]
加我微信，拉你进群
![IMG_1574](https://user-images.githubusercontent.com/21151126/222130177-a85c1e2b-bdaa-4c96-98bc-2c4c8da02615.JPG)

**Comment by hechao-ustc** [comment]
加我微信：superme9603，拉你进群

**Comment by liujiwen-up** [comment]
加我微信：Ljw919666，我拉你进群交流一波

**Comment by echo-hhj** [comment]
加我微信：hhj_0530，拉你进群交流
